apps/api/src/billing/repositories/user-wallet/user-wallet.repository.ts

```diff
@@ -60,7 +60,7 @@ export class UserWalletRepository {
 
   async find(query?: Partial<DbUserWalletOutput>) {
     const fields = query && (Object.keys(query) as Array<keyof DbUserWalletOutput>);
-    const where = fields.length ? and(...fields.map(field => eq(this.userWallet[field], query[field]))) : undefined;
+    const where = fields?.length ? and(...fields.map(field => eq(this.userWallet[field], query[field]))) : undefined;
```

Comment:
Drizzle recently added a [.if()](https://orm.drizzle.team/learn/latest-releases/drizzle-orm-v03010#-if-function-added-to-all-where-expressions) method for conditional where like this. Not necessary to use it, just sharing since it's missing from the doc.